tests/osaka/eip7939_count_leading_zeros/test_count_leading_zeros.py

```diff
@@ -276,3 +278,95 @@ def test_clz_jump_operation(
         post[callee_address] = Account(storage={"0x00": expected_clz})
 
     state_test(pre=pre, post=post, tx=tx)
+
+
+@pytest.mark.valid_from("Osaka")
+@pytest.mark.parametrize("bits", [0, 16, 64, 128, 255])
```

Comment:
Same comment as in the other EIP, I think we should have these bits inside of the same test case just to reduce the number of generated tests.